.github/workflows/installers.yml

```diff
@@ -69,14 +69,20 @@ jobs:
         python -m pip install --upgrade pip
         python -m pip install wheel setuptools
 
-        python -m pip install numpy scipy docutils "pytest<6" sphinx unittest-xml-reporting
+        python -m pip install numpy scipy==1.7.3 docutils "pytest<6" sphinx unittest-xml-reporting
         python -m pip install tinycc h5py sphinx pyparsing html5lib reportlab pybind11 appdirs
         python -m pip install six numba mako ipython qtconsole xhtml2pdf unittest-xml-reporting pylint
-        python -m pip install qt5reactor periodictable PyQt5 uncertainties debugpy
+        python -m pip install qt5reactor periodictable uncertainties debugpy
 
-    - name: Install lxml and MPL (OSX)
+    - name: Install PyQt (Windows + Linux)
```

Comment:
You are installing PyQt twice on mac and never on win and linux -- thus SasView installer fails to load.  I think line 77 should not change and the changes below moved here then add the windows/linux PyQt .. just make sure that the command is `!=` rather than `== masos-latest` :smile

Same comment applies to the release yml changes

Reply:
Good catch @butlerpd! Indeed it was not what I inteded to do... It's been fixed now.